Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextStringTests.cs

```diff
@@ -0,0 +1,148 @@
+
+            Assert.IsType<string>(result);
+            Assert.Equal(Zero, result.Length);
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_WithLenght100Chars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_OfAlphabeticChars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred, MoreDotNet.Models.CharType.AlphabeticAny);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+            for (int i = 0; i < OneHundred; i++)
+            {
+                Assert.True(char.IsLetter(result[i]));
+            }
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_OfAlphabeticLowerChars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred, MoreDotNet.Models.CharType.AlphabeticLower);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+            for (int i = 0; i < OneHundred; i++)
+            {
+                Assert.True(char.IsLower(result[i]));
+            }
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_OfAlphabeticUpperChars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred, MoreDotNet.Models.CharType.AlphabeticUpper);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+            for (int i = 0; i < OneHundred; i++)
+            {
+                Assert.True(char.IsUpper(result[i]));
+            }
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_OfAlphanumericChars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred, MoreDotNet.Models.CharType.AlphanumericAny);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+            for (int i = 0; i < OneHundred; i++)
+            {
+                Assert.True(char.IsLetterOrDigit(result[i]));
+            }
+        }
+
+        [Fact]
+        public void NextString_ShouldReturnString_OfAlphanumericLowerChars()
+        {
+            var random = new Random();
+            var result = random.NextString(OneHundred, MoreDotNet.Models.CharType.AlphanumericLower);
+
+            Assert.IsType<string>(result);
+            Assert.Equal(OneHundred, result.Length);
+            for (int i = 0; i < OneHundred; i++)
+            {
+                var currentChar = result[i];
+                Assert.True((currentChar >= 48 && currentChar < 58) || (currentChar >= 97 && currentChar < 123));
```

Comment:
Magic numbers - please extract the check into a meaningful method. Or even write a new extension method?